examples/multipart-upload-wrapper-example/README.md

```diff
@@ -0,0 +1,7 @@
+# An example web app shows how to use Here Data SDK for TypeScript to upload and publish large data in a browser.
+
+This is an HTML+JS UI and HTTP server with mocked routes to simulate the real DataStore.
```

Comment:
What are "mocked routes"?

Reply:
Mocked Blob and Publish endpoints and responses to simulate the real OLP

Reply:
lines 47 - 118 https://github.com/heremaps/here-data-sdk-typescript/pull/460/files#diff-52da086ca44fc827896c5ba8c3e6d147db75e34b8abfa7dfa0c090316f9544c7R47